docs/layouts/shortcodes/databases-passwords.md

```diff
@@ -0,0 +1,23 @@
+## Password generation
+ 
+When you connect your app to a database,
+an empty password is generated for the database by default.
+This can cause issues with your app.
+ 
+To generate real passwords for your database,
+define custom endpoints in your [service configuration](#1-configure-the-service).
+For each custom endpoint you create,
+you get an automatically generated password,
+similarly to when you create [multiple databases](#multiple-databases).
+Note that you can't customize these automatically generated passwords.
+
+After your custom endpoints are exposed as relationships in your [app configuration](../../create-apps/_index.md),
+you can retrieve the password for each endpoint
+through the `PLATFORM_RELATIONSHIPS` [environment variable](../../development/variables/use-variables.md#use-platformsh-provided-variables).
+You can access the `PLATFORM_RELATIONSHIPS` environment variable directly [in your app](../../development/variables/use-variables.md#access-variables-in-your-app),
+for example using a Platform.sh Configuration Reader.
```

Comment:
:clipboard: All of the libraries are called <LANGUAGE_NAME> Config Reader. So if you're going for a proper noun here, as it seems with the capitals, it should be the shorter form.
```suggestion
for example using a Platform.sh Config Reader.
```

Reply:
Otherwise, it can be non-proper, since it's not the full name
```suggestion
for example using a Platform.sh configuration reader.
```